docs/advanced_topics/ui.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: default
-nav_order: 2
+nav_order: 50
 parent: Advanced topics
-title: Custom UI
+title: "User Configuration - <span class=\"nowrap\">Custom UI</span>"
```

Comment:
User configuration - <span class=\"nowrap\">custom UI</span>

Adjusting the capitalization looks questionable in this case, but this way it would be consistent.